goprocess/gp.go

```diff
@@ -35,10 +35,13 @@ func FindAll() []P {
 	var wg sync.WaitGroup
 	wg.Add(len(pss))
 	found := make(chan P)
+	limitCh := make(chan struct{}, 10)
```

Comment:
How was the number `10` chosen? Might make sense to add a `const` with a documenting comment.

Reply:
Just because this number is small enough(to avoid EMFILE), and limit concurrenct tasks to 10 will not take obviously more time than no limitation(tested on my mac, for about 400 processes)